### ISSUE TITLE
Snooze or skip alarm, with a reason

### ISSUE THREAD

**Issue by andreoliwa** [andreoliwa/python-dontforget#7]
Do this one after #47, #45 and #44.

Some snooze reasons:
- [ ] I don't want to do it now
- [ ] I can't do it now
- [ ] I'm working now

Skip reasons:
- [ ] There is no need to do this now

Reasons:
- [ ] Create a `Reason` model with columns: id, title, can_snooze (bool), can_skip (bool)
- [ ] A reason can be used on snooze, on skip, or on both; at least of the flags must be set

Telegram bot:
- [ ] Show buttons with reasons to skip or snooze the chore
- [ ] Accept new text as a different reason
- [ ] Examples of reasons: busy, working, don't want to, not needed now, not in the mood

**Comment by andreoliwa** [comment]
This issue has become obsolete. See #135.
